cmd/install/install.go

```diff
@@ -160,11 +161,13 @@ func NewCommand() *cobra.Command {
 	opts.MetricsSet = metrics.DefaultMetricsSet
 	opts.EnableConversionWebhook = true // default to enabling the conversion webhook
 	opts.ExternalDNSImage = ExternalDNSImage
+	opts.HCDestroyGracePeriod = 0
 
 	cmd.PersistentFlags().StringVar(&opts.Namespace, "namespace", "hypershift", "The namespace in which to install HyperShift")
 	cmd.PersistentFlags().StringVar(&opts.HyperShiftImage, "hypershift-image", version.HyperShiftImage, "The HyperShift image to deploy")
 	cmd.PersistentFlags().BoolVar(&opts.Development, "development", false, "Enable tweaks to facilitate local development")
 	cmd.PersistentFlags().BoolVar(&opts.EnableDefaultingWebhook, "enable-defaulting-webhook", false, "Enable webhook for defaulting hypershift API types")
+	cmd.PersistentFlags().DurationVar(&opts.HCDestroyGracePeriod, "destroy-grace-period", opts.HCDestroyGracePeriod, "Grace period for HostedCluster to be deleted. The format is duration; e.g. 30s or 1h30m45s; 0 means no timeout; default = 0")
```

Comment:
It doesn't hurt anything to have it, but you don't necessarily have to include default in the note field. If the value is defaulted, cobra CLI will show the default when a user uses the `--help` flag.

Reply:
You're right, ty for heads up